CHANGELOG.md

```diff
@@ -86,6 +86,7 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 - console: add `<3 hasura` section to view updates and notifications from Hasura (#5070)
 - docs: add docs page on networking with docker (close #4346) (#4811)
 - docs: add postgres concepts page to docs (close #4440) (#4471)
+- docs: add reset metadata guide to docs (close #3874) (#5961)
```

Comment:
We don't need a changelog entry for this. I added the `no-changelog-required` label. Please remove this.